src/mappings/services/poolFeeService.ts

```diff
@@ -63,32 +63,45 @@ export class PoolFeeService extends PoolFee {
     logger.info(`Removing PoolFee ${data.feeId}`)
     const { poolId, feeId } = data
     const poolFee = await this.get(`${poolId}-${feeId}`)
-    if(!poolFee) throw new Error('Unable to remove PoolFee. PoolFee does not exist.')
+    if (!poolFee) throw new Error('Unable to remove PoolFee. PoolFee does not exist.')
     poolFee.isActive = false
     return poolFee
   }
 
   public charge(data: Omit<PoolFeeData, 'amount'> & Required<Pick<PoolFeeData, 'amount'>>) {
     logger.info(`Charging PoolFee ${data.feeId} with amount ${data.amount.toString(10)}`)
-    if(!this.isActive) throw new Error('Unable to charge inactive PolFee')
+    if (!this.isActive) throw new Error('Unable to charge inactive PolFee')
     this.sumChargedAmount += data.amount
     this.sumChargedAmountByPeriod += data.amount
```

Comment:
We should increase pending here:
```suggestion
    this.sumChargedAmountByPeriod += data.amount
    this.pendingAmount += data.amount
```